src/flowMC/resource/kernel/NF_proposal.py

```
@@ -99,7 +137,18 @@ def sample_flow(
         self,
         rng_key: Key,
         n_steps: int,
-    ):
+    ) -> tuple[Float[Array, "n_steps n_dim"], Float[Array, " n_steps"]]:
```

Ruff (0.15.9)

[error] Syntax error in forward annotation: Unexpected token at the end of an expression

(F722)